Objects/typeslots.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-# Usage: typeslots.py < Include/typeslots.h typeslots.inc
+# Usage: ./python typeslots.py < Include/typeslots.h typeslots.inc
```

Comment:
This looks wrong; either you're in the directory with `./python` or in `Objects`.
Same for `_sha3/cleanup.py`. Did you try these commands?

Reply:
> This looks wrong; either you're in the directory with ./python or in Objects.
Oh, you are right. I just point developer should use `./python` in here. Using entire command would be better.

> Same for _sha3/cleanup.py. Did you try these commands?
the command in _sha3/cleanup.py is a entire command.

Reply:
There have an another question. Typeslots.py should be moved to the dir of Tools? It's weird leave it alone in Objects. Maybe we should open an  new bpo to discuss it.